src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -5106,15 +5106,10 @@ void AuraEffect::HandleAuraDummy(AuraApplication const* aurApp, uint8 mode, bool
                 case 46354:                                     // Blood Elf Illusion
                     if (caster)
                     {
-                        switch (caster->getGender())
-                        {
-                            case GENDER_FEMALE:
-                                caster->CastSpell(target, 46356, true, nullptr, this);
-                                break;
-                            case GENDER_MALE:
-                                caster->CastSpell(target, 46355, true, nullptr, this);
-                                break;
-                        }
+                        if (caster->getGender() == GENDER_FEMALE)
+                            caster->CastSpell(target, 46356, this);
+                        else
+                            caster->CastSpell(target, 46355, this);
```

Comment:
can you add comments to these spells, so we know what they are

Reply:
Yes. Got a switch error in the workflow dealing with not having a gender neurtral option or default.
Ill deal with it next week or two.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/14843/commits/c5add62567a2f1850552da69f846a7796eb94b28